acra/configuring-maintaining/installing/installing-from-repository.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Installing Acra from the Cossack Labs repository
+bookCollapseSection: true
+---
+
+### Installing Acra from the Cossack Labs repository
+
+> Note: A quick reminder - the server-side Acra components are not compatible with Windows OS as host OS, consider [using Docker]({{< ref "acra/guides/trying-acra-with-docker/" >}}).
+
+#### Debian / Ubuntu
+
+**1. Import the public key used by Cossack Labs to sign packages:**
+
+```console
+wget -qO - https://pkgs.cossacklabs.com/gpg | sudo apt-key add -
+```    
+
+> Note: If you wish to validate the key fingerprint, it is: `29CF C579 AD90 8838 3E37 A8FA CE53 BCCA C8FF FACB`.
+
+**2. You may need to install the `apt-transport-https` package before proceeding:**
+
+```console
+sudo apt-get install apt-transport-https
+```    
+
+**3. Add the Cossack Labs repository to your `sources.list`.**        
+You should add a line that specifies your OS name and the release name:
+
+```console
+deb https://pkgs.cossacklabs.com/stable/$OS $RELEASE main
+```    
+
+* `$OS` should be `debian` or `ubuntu`.
+* `$RELEASE` should be one of Debian or Ubuntu release names. You can determine this by running `lsb_release -cs`, if you have `lsb_release` installed.
+
+We currently build packages for the following OSs and RELEASE combinations:
+
+- *CentOS 7*,
+- *Debian "Jessie" (Debian 8)*,
+- *Debian "Stretch" (Debian 9)*,
+- *Ubuntu Xenial Xerus (Ubuntu 16.04)*,
+- *Ubuntu Artful Aardvark (Ubuntu 17.10)*,
+- *Ubuntu Bionic Beaver (Ubuntu 18.04)*.
```

Comment:
Blatant lies. The current list is this:

```markdown
- `debian jessie` – Debian 8 (Jessie)
- `debian stretch` – Debian 9 (Stretch)
- `ubuntu xenial` – Ubuntu 16.04 LTS (Xenial Xerus)
- `ubuntu bionic` – Ubuntu 18.04 LTS (Bionic Beaver)
```

I believe CentOS 7 should not be here, in a section about installation on Debian.

Also, it would be nice to provide packages for Debian 10, soon-to-be-released Debian 11, and Ubuntu 20.04. It's 2021, after all.

Reply:
Updated to 
```
- *Debian "Wheezy" (Debian 7)*,
- *Debian "Jessie" (Debian 8)*,
- *Debian "Stretch" (Debian 9)*,
- *Debian "Buster" (Debian 10)*,
- *Trusty Precise Pangolin (Ubuntu 12.04)*.
- *Trusty Trusty Tahr (Ubuntu 14.04)*.
- *Ubuntu Xenial Xerus (Ubuntu 16.04)*,
- *Ubuntu Yakkety Yak (Ubuntu 16.10)*.
- *Ubuntu Zesty Zapus (Ubuntu 17.04)*.
- *Ubuntu Artful Aardvark (Ubuntu 17.10)*,
- *Ubuntu Bionic Beaver (Ubuntu 18.04)*.
- *Ubuntu Focal Fossa (Ubuntu 20.04)*.
```
Checked versions from https://pkgs-ce.cossacklabs.com/stable/debian/dists/ and https://pkgs-ce.cossacklabs.com/stable/ubuntu/dists/

Reply:
`dists` is only a list of available distributions in the repository. A package may be available only for some of them. Specifically, `acra` package is currently available only for the ones I listed: Debian 8 & 9, Ubuntu 16.04 & 18.04.

Reply:
How you check it? You are right. I looked on artful and found in `Packages` package `acra`. But can't download .deb *(
And your list is correct.